app/views/password_reset_mailer/password_reset.text.erb

```diff
@@ -0,0 +1,14 @@
+Hello <%= @user.first_name %>,
```

Comment:
I think all the one's we're referencing are required by the Chef server so the chance is probably pretty low.